src/iop/grain.c

```diff
@@ -452,6 +512,14 @@ static void strength_callback(GtkWidget *slider, gpointer user_data)
   dt_dev_add_history_item(darktable.develop, self, TRUE);
 }
 
+static void midtones_bias_callback(GtkWidget *slider, gpointer user_data)
+{
+  dt_iop_module_t *self = (dt_iop_module_t *)user_data;
+  if(self->dt->gui->reset) return;
+  dt_iop_grain_params_t *p = (dt_iop_grain_params_t *)self->params;
+  p->midtones_bias = dt_bauhaus_slider_get(slider) / 100;
```

Comment:
Also, this does not seem nice, it is better to make `p->midtones_bias` contain the same scale as the slider, and *if needed*, let the `commit_params()` do the scaling.